fuse/ntfs-3g/Portfile

```diff
@@ -1,9 +1,8 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-
```

Comment:
Why removing this line?

Reply:
The ports I maintain usually don't have that blank line. Force of the habit.